### PR TITLE
Config file for CORS

### PR DESCRIPTION
**Pull request by pizzaluc**
#se include a summary of the change and which issue it fixes. Please also include relevant motivation and context. List any dependencies that are required for this change. -->

Add a config file with some default values for CORS (some redundant ones too)

# Type of change

Please check all boxes that apply to this change below.

| Type                     | Applies ✅|
| ------------------------ | :-------: |
| Backend                  |     ✅     |
| Frontend                 |          |
| Github/Ci/Admin          |          |
| Documentation required   |          |

# Checklist

Please follow the checklist below and check all completed items as you work on the PR.

I have..
- [x] performed a self-review of my own code
- [ ] commented my code, particularly in hard-to-understand areas
- [ ] written documentation
- [ ] added automated tests
- [ ] merged dependencies to `develop

### CODE REVIEW

File: backend/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "CORS": {
+        "AllowedHosts": [ "*", "localhost" ]
```
Comment (Maasdamind) [inline backend/appsettings.json:3]: Same question as the other appsettings. Also, why are there two appsettings files?

**Review by Maasdamind — COMMENTED** [review]
Looks Great! Just have those two questions

File: backend/appsettings.json (hunk 1 above)
Comment (pizzaluc) [reply backend/appsettings.json]: removed the duplicate one

File: appsettings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "CORS": {
+        "AllowedHosts": [ "*", "localhost", "example.com" ]
```
[the remark this answers is not in the document]
Comment (pizzaluc) [reply appsettings.json]: added the correct local address, so its not open to the whole internet

**Review by tjomson — APPROVED** [review]
It has been open for so long, and the questions have been resolved, so i guess it is good

**Pull request merged**